src/components/Home.js

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
-import Header from './Header';
-import Body from './Body';
-import Footer from './Footer';
+import React from "react";
+//Components
```

Comment:
**Unnecessary comment**, once again.

It is  a good practise to have a comment where things happen (business logic/services), **stating obvious is not helping anyone**.

Reply:
Yes, unnecesary commentary once again.